### PR TITLE
AEA-270: Deploy coordinator to ECS

### PR DESCRIPTION
**Pull request by mfjarvis**
…during the CI build.

## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -39,7 +50,7 @@ jobs:
       - bash: "sudo apt update && sudo apt-get install --yes default-jre default-jdk"
         displayName: Install apt dependencies
 
-      - bash: "wget -q -O - https://dl.google.com/linux/linux_signing_key.pub | sudo apt-key add -; curl -sL https://deb.nodesource.com/setup_13.x | sudo -E bash -; sudo apt-get install -y nodejs"
+      - bash: "wget -q -O - https://dl.google.com/linux/linux_signing_key.pub | sudo apt-key add -; curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -; sudo apt-get install -y nodejs"
```
Comment (conner-jowitt) [inline azure-pipelines.yml:53]: Why did we go down a version of node?

File: ecs-proxies-deploy.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#docker_service_cpu: 256
+#docker_service_memory: 512
+
+docker_service:
+    - name: eps-coordinator
+      expose: true
+      environment:
+          - name: NODE_ENV
+            value: production
```
Comment (conner-jowitt) [inline ecs-proxies-deploy.yml:9]: What does this change other than the log level? Should it be set to internal-dev?

File: proxies/live/apiproxy/policies/AssignMessage.AddCors.xml [hunk 3]
```diff
@@ -4,9 +4,9 @@
     <Set>
         <Headers>
             <Header name="Access-Control-Allow-Origin">{request.header.origin}</Header>
-            <Header name="Access-Control-Allow-Headers">origin, x-requested-with, accept, content-type, nhsd-session-urid</Header>
+            <Header name="Access-Control-Allow-Headers">origin, x-requested-with, accept, content-type, if-match, nhsd-session-urid</Header>
             <Header name="Access-Control-Max-Age">3628800</Header>
-            <Header name="Access-Control-Allow-Methods">GET, PUT, POST, DELETE</Header>
+            <Header name="Access-Control-Allow-Methods">GET, PUT, POST, PATCH, DELETE</Header>
```
Comment (conner-jowitt) [inline proxies/live/apiproxy/policies/AssignMessage.AddCors.xml:9]: Why did we add PATCH?

File: Makefile [hunk 4]
```diff
@@ -64,7 +66,8 @@ test:
 	&& export API_TEST_ENV_FILE_PATH=$(or $(API_TEST_ENV_FILE_PATH),tests/e2e/environments/local.postman_environment.json) \
 	&& export API_TEST_URL=$(or $(API_TEST_URL),localhost:9000) \
 	&& npm run test
-	cd sandbox && npm t
+	cd sandbox && npm t && cd ..
```
[the remark this answers is not in the document]
Comment (mfjarvis) [reply Makefile:69]: Good to know, will remove

File: ecs-proxies-deploy.yml (hunk 2 above)
Comment (mfjarvis) [reply ecs-proxies-deploy.yml:9]: Different libraries can do what they want with it. The only options are "development" and "production" though, so I think everything other than your local machine is "production".

File: proxies/live/apiproxy/policies/AssignMessage.AddCors.xml (hunk 3 above)
Comment (mfjarvis) [reply proxies/live/apiproxy/policies/AssignMessage.AddCors.xml:9]: Copied the proxies over from PDS. Will change to just GET and POST, as that's all we're using for the moment.

Comment (mfjarvis) [reply proxies/live/apiproxy/policies/AssignMessage.AddCors.xml:9]: Same goes for the if-match header, we're not using that (yet)

**Pull request merged**